main/lsp/request_dispatch.cc

```diff
@@ -312,165 +23,52 @@ void LSPLoop::processRequest(std::unique_ptr<LSPMessage> msg) {
 }
 
 void LSPLoop::processRequests(vector<unique_ptr<LSPMessage>> messages) {
```

Comment:
This is the code path that single threaded code, like sorbet.run, takes.

We don't need a QueueState since both the LSPPreprocessor and LSPLoop share ownership of it.